src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -20,6 +20,11 @@ public class ObjectResult : ActionResult
 
         public Type DeclaredType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the HTTP status code.
+        /// </summary>
+        public int? StatusCode { get; set; }
```

Comment:
know this PR is closed but it looks like we're missing a change to `NegotiatedContentResult` due to this added property.  new warnings when building the WebApiCompatShim project:
```
...\src\Microsoft.AspNet.Mvc.WebApiCompatShim\ActionResults\NegotiatedContentResult.cs(31,31):
warning CS0108: 'NegotiatedContentResult<T>.StatusCode' hides inherited member 'ObjectResult.StatusCode'.
Use the new keyword if hiding was intended.
```